src/Serilog.Enrichers.Environment/project.json

```diff
@@ -0,0 +1,24 @@
+﻿{
+  "version": "2.0.0-beta-*",
+  "description": "The enviornment enrichers for Serilog.",
+  "authors": [ "Serilog Contributors" ],
+  "tags": [ "serilog", "machine", "enricher" ],
+  "projectUrl": "http://serilog.net",
+  "licenseUrl": "http://www.apache.org/licenses/LICENSE-2.0",
+  "iconUrl": "http://serilog.net/images/serilog-extension-nuget.png",
```

Comment:
Should use:

```
http://serilog.net/images/serilog-enricher-nuget.png
```

![Rendered](http://serilog.net/images/serilog-enricher-nuget.png)